### ISSUE TITLE
Upgrade TailwindCSS for the latest

### ISSUE THREAD

**Issue by kalicki** [efellemedia/proton#22]
TailwindCSS from v0.7.2 to v1

Ref: https://tailwindcss.com/docs/upgrading-to-v1/

**Comment by kaidesu** [comment]
Thinking I might extract the tailwind styles out to a separate package for users to pull in/reference. Would be easier to maintain separately that way.

e.g. `proton-tailwind-theme`

**Comment by kalicki** [comment]
Right! however the proton in its compontens injects classes based in classes by Tailwind, but need change class eg: `text-grey-darker` to `text-gray-700` or something more generic
